components/VoteHistoryTable/VoteHistoryTable.tsx

```diff
@@ -0,0 +1,154 @@
+import { green, grey500, red500 } from "constants/colors";
+import { formatNumberForDisplay } from "helpers";
+import { usePanelContext } from "hooks";
+import { CSSProperties } from "react";
+import styled from "styled-components";
+import { VoteT } from "types";
+
+export function VoteHistoryTable({ votes }: { votes: VoteT[] }) {
```

Comment:
Component for showing the user's voting activity and calculated cumulative slash for each vote. Shows all grey (empty) for votes where the user did nothing.